addon/debounce-task.ts

```diff
@@ -52,7 +52,7 @@ const registeredDebounces: IMap<
    @param { Object } obj the instance to register the task for
    @param { String } name the name of the task to debounce
    @param { ...* } debounceArgs arguments to pass to the debounced method
-   @param { Number } wait the amount of time to wait before calling the method (in milliseconds)
+   @param { Number } spacing the amount of time to wait before calling the method (in milliseconds)
    @public
```

Comment:
Since we support checking for `immediate`, we should likely document it. Can we add this?